### PR TITLE
Purchases: Add more testing platforms

### PR DESCRIPTION
**Pull request by dbspringer**
Simple update to release survey to more platforms to test on. Please refer to https://github.com/Automattic/wp-calypso/pull/4838 for full testing instructions.

Test live: https://calypso.live/?branch=update/remove-purchase-survey-platforms

### CODE REVIEW

File: config/stage.json [hunk 1]
```diff
@@ -94,7 +94,8 @@
 		"upgrades/domain-search": true,
 		"upgrades/in-app-purchase": false,
 		"upgrades/paypal": true,
-		"upgrades/premium-themes": true
+		"upgrades/premium-themes": true,
+		"upgrades/removal-survey": true
```
Comment (gwwar) [inline config/stage.json]: I would suggest holding off on enabling stage until we're also ready to turn a feature on in prod.

Comment (dbspringer) [inline config/stage.json]: No problemo, I'll just do wpcalypso and horizon.

**Comment by dbspringer** [comment]
Removed the config value from stage.json. Just going w/ wpcalypso & horizon for now.

**Comment by gwwar** [comment]
After completing the survey I see:
screen shot 2016-06-29 at 9 43 12 am

instead of placeholder loaders. This isn't blocking, but lets tidy this up before this is turned on in prod.

**Comment by dbspringer** [comment]
@gwwar I think that's the current remove product flow outside of any updates I've made. But I agree, I'll take a look at making it a cleaner transition while I'm at it. 

Thanks! 🙇

**Pull request merged**